csrc/dynamic_transform.cpp

```diff
@@ -543,10 +564,16 @@ std::string DynamicTransformConcretizationInfo::toString() const {
   NVF_ERROR(
       reshape_transforms_.size() ==
       initial_info_->getDynamicReshapedTensorViews().size());
-  for (const auto& [tv_index, analyze_result] : reshape_transforms_) {
+  for (const auto& [tv_index, view_info] : reshape_transforms_) {
     auto tv = initial_info_->getDynamicReshapedTensorViews().at(tv_index);
-    ss << indent << indent << tv->toString() << " (index=" << tv_index << "), "
-       << analyze_result.toString() << "\n";
+    if (std::holds_alternative<AnalyzeViewResult>(view_info)) {
+      ss << indent << indent << tv->toString() << " (index=" << tv_index
```

Comment:
Nit: this may just be legacy code. I saw `indent(ss, indent_size)` used more often, which seems to be simpler than `ss << indent << indent`.

Reply:
You're right. It's legacy code but I like that style better. I'll change it in a separate PR.